packages/@okta/vuepress-site/docs/reference/releases-at-okta/index.md

```diff
@@ -1,71 +1,72 @@
+| In production orgs         | &cross;              | By request or self-service, depending on the feature| &check;| &check; |
+| Documentation              | Limited              | &check;              | &check;              | NA         |
 
-## Beta
+## Beta Release
 
-Features in Beta are managed and supported by the Product Team and have been internally validated for a set of use cases.
-Minimal documentation is supplied for API-related Beta releases; API endpoint and configuration information is usually supplied.
+Features in Beta are managed and supported by the Product Team and have been internally validated for a set of use cases. Minimal documentation is supplied to customers for API-related Beta releases, and Beta documentation isn't available at developer.okta.com. API endpoint and configuration information is usually supplied.
 
-> **Important:** Okta only enables Beta features in Preview org environments, because features in Beta aren't supported by Okta Customer Support and
-may change at any time during the Beta release.
+> **Important:** Okta only enables Beta features in Preview org environments because features in Beta aren't supported by Okta Customer Support and may change at any time during the Beta release.
 
-Okta selects a small number of customers for early testing of features in a Beta release.
-Customers participating in a Beta program agree to provide feedback which is required for maturing the feature;
-however, the timeline for addressing specific areas of feedback (including bugs) is determined by Okta's market requirements for that feature.
-Any customer interested in a feature in Beta must apply by visiting [our Beta Signup page](https://support.okta.com/help/OktaBetaProgramHome).
-Only customers with use cases that match our use cases are invited.
+Okta selects a small number of customers for early testing of features in a Beta release. Customers participating in a Beta program agree to provide feedback that is required for maturing the feature. However, the timeline for addressing specific areas of feedback (including bugs) is determined by Okta's market requirements for that feature.
 
-Beta releases are either high touch or low touch:
+Beta releases are managed in two ways:
 
-* High-touch Beta releases involve regular contact with Okta, typically consisting of conference calls covering specified use cases, deployment guidance, and feedback.
-* Low-touch Beta releases are self-directed without ongoing support and limited to collecting feedback.
+* **Beta:** Involves regular contact with Okta, which may include conference calls between you and Okta that cover specific use cases, deployment guidance, and feedback. Betas are also announced to customers through the Ozone newsletter and Communities. Okta may also solicit customers to participate in a Beta via email.
 
-Documentation for features in Beta release is marked with the Beta icon: <ApiLifecycle access="beta" />
```

Comment:
Want to confirm it's okay to remove mention of the Beta lifecycle banner: have we stopped using it on the site?

Reply:
We haven't stopped, but the only instances of it are legacy pages that I am actively pushing to get rid of it.